3회차/윤효근/0_민석이의과제체크하기.py

```diff
@@ -1,3 +1,18 @@
 import sys
 
 sys.stdin = open("_민석이의과제체크하기.txt")
+
+T = int(input())
+result=[]
+for test_case in range(1, T + 1):
+    n,k = map(int,input().split())
+    x = list(map(int,input().split()))
+    for i in range(1,n+1):
+        result.append(i)
+    for i in range(len(x)):
+        result.remove(x[i])
```

Comment:
좋은 풀이네요! 👍 
`remove`를 사용하지 않는 풀이도 있습니다 😄 
```suggestion
    for i in range(1,n+1):
        if i not in x:
            result.append(i)
```